go/exocom-mock/exocom-mock.go

```diff
@@ -37,14 +38,18 @@ func (exoCom *ExoComMock) Listen(port int) error {
 	return exoCom.server.ListenAndServe()
 }
 
-// messageHandler is called when the given socket receives a message.
-// It logs the received message.
-func (exoCom *ExoComMock) messageHandler(socket *websocket.Conn) {
-	message, err := exoCom.readMessage(socket)
-	if err != nil {
-		panic(err)
+// websocketHandler is called when the given socket connects
+func (exoCom *ExoComMock) websocketHandler(socket *websocket.Conn) {
+	for {
+		message, err := exoCom.readMessage(socket)
+		if err == io.EOF {
+			break
+		} else if err != nil {
+			panic(err)
```

Comment:
Do we really want to crash the entire running test here? What about printing the error and just breaking the loop? I can see such errors happening when ramping down tests.